### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by brahimdidi**
In this Project I will set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules.

Requirements
This is the first time I will use Gitflow.
Create an npm project with webpack.
Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Leaderboard-project",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (MrBrN197) [inline package.json]: - [ ] Hi, please make sure to add scripts for running the project in your package.json file. add at the very list a `start` script for webpack to run.
## Something like this
```suggestion
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  },
```

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+node_modules
+.vscode
+hint-report
```
Comment (MrBrN197) [inline .gitignore]: - [ ] [OPTIONAL] Your hint-report folder is still in your branch. please delete the folder then run this
`git rm -r --cached hint-report` to remove it from being tracked.

File: dist/index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+<script defer src="bundle.js"></script></head>
+<body>
+    <h1 class="title">Leaderboard</h1>
+    <div class="container">
+    <div class="list-container">
+        <div class="rs-div"> <h2 class="recent-scores">Recent  scores</h2><button class="refresh">refresh</button></div>
```
Comment (MrBrN197) [inline dist/index.html]: - [ ] [OPTIONAL] you can add some padding to your refresh button. currently, it's is too close to the left & right edges.
image

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @brahimdidi 👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi @brahimdidi 👋, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**